package.json

```diff
@@ -21,8 +21,12 @@
     "atom": ">=0.174.0"
   },
   "dependencies": {
-    "atom-linter": "^3.0.0"
+    "atom-linter": "^3.0.0",
+    "atom-package-deps": "^3.0.0"
```

Comment:
[greenkeeper.io](http://greenkeeper.io/) will yell as soon as it sees the out of date version anyway :wink:.